src/compiler/factory/nodeTests.ts

```diff
@@ -291,27 +291,22 @@ export function isAsteriskToken(node: Node): node is AsteriskToken {
     return node.kind === SyntaxKind.AsteriskToken;
 }
 
-/** @internal */

```

Comment:
Are these necessary? We only export a few token/keyword-specific node tests functions, and definitely don't even have some for many tokens/keywords.

Reply:
These are all tests used in `visitEachChild` so I figured they would be ones that would be wanted externally as well: [`7e6222a` (#52284)](https://github.com/microsoft/TypeScript/pull/52284/commits/7e6222a8b6f0fb19aaa40163c9d5493575fd1e71)

But, I'm happy to drop these if you think they're not going to end up being valuable.

Reply:
We can leave them in.